tools/schemapi/schemapi.py

```diff
@@ -564,7 +561,7 @@ def from_dict(
                         dct,
                         schema=possible_schema,
                         rootschema=rootschema,
-                        default_class=default_class,
+                        default_class=cls,
                     )
 
         if isinstance(dct, dict):
```

Comment:
This change needs to be applied to the copy in the package as well in order for the tests to cover it.

Reply:
I copied it over by running the `generate_schema_wrapper.py` script.